pkg/local_object_storage/blobstor/common/errors.go

```diff
@@ -1,10 +1,14 @@
 package common
 
-import "errors"
+import (
+	"errors"
+
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/util/logicerr"
+)
 
 // ErrReadOnly MUST be returned for modifying operations when the storage was opened
 // in readonly mode.
-var ErrReadOnly = errors.New("opened as read-only")
+var ErrReadOnly = logicerr.Wrap(errors.New("opened as read-only"))
```

Comment:
Maybe provide similar constructor `logicerr.New(string) error`? Otherwise, I would expect wrapping on the `return` side instead of the definition place.

Reply:
The intention was to keep the package as small as possible. But I guess `New` is not big and would save us some code.

Reply:
Also, it is intentionally used in the definition -- this way it is harder to misuse or forget about something.
This wrapper is used only for error reporting and here the _essence_ of the error matters, not where it ocurred.

Reply:
Fixde